Jellyfin.Server/Program.cs

```diff
@@ -183,15 +186,31 @@ private static async Task StartApp(StartupOptions options)
                 new ManagedFileSystem(_loggerFactory.CreateLogger<ManagedFileSystem>(), appPaths),
                 GetImageEncoder(appPaths),
                 new NetworkManager(_loggerFactory.CreateLogger<NetworkManager>()));
+
             try
             {
+                // If hosting the web client, validate the client content path
```

Comment:
something looks off with indentation here

Reply:
GitHub is hiding a few lines above that I think makes the indenting clearer
![image](https://user-images.githubusercontent.com/2626103/78131686-c1a55b80-741b-11ea-8b5e-611e378a2181.png)

Reply:
You guys know you can expand the diff on github up and down, right? ;)